tools/interactive/interactivetool_wilson.xml

```diff
@@ -10,9 +10,12 @@
     </entry_points>
     <environment_variables>
         <environment_variable name="WILSON_LANDING_PAGE">feature_selection</environment_variable>
+        <environment_variable name="WILSON_BLACKLIST_EXAMPLES">true</environment_variable>
     </environment_variables>
     <command><![CDATA[
         mkdir -p /srv/shiny-server/external_data/ &&
+        echo $WILSON_LANDING_PAGE > /home/shiny/.Renviron &&
```

Comment:
@jenzopr I'm not sure this will work. What you want inside the .Renviron is `WILSON_BLACKLIST_EXAMPLES=true` or how should it look like?

If so, I guess what we need is `echo "WILSON_LANDING_PAGE=\$WILSON_LANDING_PAGE" > /home/shiny/.Renviron &&`

Reply:
Thats true - my bad! 🙈